packages/@storefront/structure/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [patch]
+### Removed
+- SF-1255: Remove redundant invocation of `FilteredList#updateItems()`.
```

Comment:
Since you're making changes anyway, perhaps you can add a small note about this reducing updates.

Reply:
Implemented. Please review and resolve.